kratos/geometries/triangle_3d_3.h

```diff
@@ -2313,26 +2325,52 @@ template<class TPointType> class Triangle3D3
 		f = Ay*Bx - Ax*By;
 		d = By*Cx - Bx*Cy;
 
-		if (std::abs(f)<1E-10) f = 0.00;
-		if (std::abs(d)<1E-10) d = 0.00;
+		if (std::abs(f) < 1E-10) f = 0.00;
+		if (std::abs(d) < 1E-10) d = 0.00;
 
-
-		if ((f>0.00 && d >= 0.00 && d <= f) || (f<0.00 && d <= 0.00 && d >= f))
-		{
+		if ((f>0.00 && d >= 0.00 && d <= f) || (f<0.00 && d <= 0.00 && d >= f)) {
 			e = Ax*Cy - Ay*Cx;
 
-			if (f>0.00)
-			{
-				if (e >= 0.00 && e <= f) return true;
-			}
-			else
-			{
-				if (e <= 0.00 && e >= f) return true;
+			if (f > 0.0) {
+				if (e >= 0.0 && e <= f) return true;
+			} else {
+				if (e <= 0.0 && e >= f) return true;

```

Comment:
```suggestion
                        if (f > 0.0) {
                            if (e >= 0.0 && e <= f) return true;
                        } else {
                            if (e <= 0.0 && e >= f) return true;
```
(minor indentation aesthetic)